Cut the 2.14.0-rc.0 release

csmarchbanks opened the pull request:
Cut the first 2.14.0 release candidate. It is possible I will add one more PR to this release, but I want to get any comments sooner than later.

brian-brazil commented on CHANGELOG.md:
```diff
@@ -1,3 +1,20 @@
+  * Not all pages are implemented yet.
+* [FEATURE] Status: Cardinality statistics added to the Runtime & Build Information page. #6125
+* [ENHANCEMENT] Remote write: fix delays in remote write after a compaction. #6021
+* [ENHANCEMENT] Service discovery: errors are no longer logged on context cancellation. #6116, #6133
```
Bugfix?

brian-brazil commented on CHANGELOG.md:
```diff
@@ -1,3 +1,20 @@
+* [BUGFIX] Promtool: Remove false duplicate rule warnings when checking rule files with alerts. #6270
+* [BUGFIX] Remote write: restore use of deduplicating logger in remote write. #6113
+* [BUGFIX] Remote write: do not reshard when unable to send samples. #6111
+* [BUGFIX] Service discovery: fix custom-sd adapter on windows. #6217
```
This isn't part of the prometheus release

csmarchbanks commented:
Thanks for the review, I have updated this for the comments, and rebased to include #6279. Right now I have 6279 listed as security/bugfix, is that ok or should it just be bugfix?

brian-brazil commented:
:+1: 

security/bugfix and on top is right, as it's basically a reflected XSS.

csmarchbanks commented:
/benchmark 2.13.1

csmarchbanks commented:
/benchmark cancel

Benchmark looks good.
